boards/arm/we_proteus3ev_nrf52840/doc/index.rst

```diff
@@ -0,0 +1,206 @@
+:ref:`build_an_application` and :ref:`application_run` for more details on
+building and running.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board Proteus-III-EV
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: we_proteus3ev_nrf52840
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+
+Testing the LEDs and buttons in the Proteus-III-EV
+**************************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+.. code-block:: console
+
+   samples/basic/blinky
+   samples/basic/button
+
+You can build and flash the examples to make sure Zephyr is running correctly on
+your board. The button and LED definitions can be found in
+:zephyr_file:`boards/arm/we_proteus3ev_nrf52840/we_proteus3ev_nrf52840.dts`.
+
+Using UART1
+***********
+
+The following approach can be used when an application needs to use
+more than one UART for connecting peripheral devices:
+
+1. Add devicetree overlay file to the main directory of your application:
+
+   .. code-block:: console
+
+      $ cat we_proteus3ev_nrf52840.overlay
+      &uart1 {
+        compatible = "nordic,nrf-uarte";
+        current-speed = <115200>;
+        status = "okay";
+        tx-pin = <14>;
+        rx-pin = <16>;
+      };
+
+   In the overlay file above, pin P0.16 is used for RX and P0.14 is used for TX
+
+2. Use the UART1 as ``device_get_binding(DT_LABEL(DT_NODELABEL(uart1)))``
+
+See :ref:`set-devicetree-overlays` for further details.
+
+Selecting the pins
+==================
+To select the pin numbers for tx-pin and rx-pin:
+
+.. code-block:: console
+
+   tx-pin = <pin_no>
+
+Open the `nRF52840 Product Specification`_, and check for the pins marked as
+'General purpose I/O'.  Note that pins marked as 'low frequency I/O only' can only be used
+in under-10KHz applications. They are not suitable for 115200 speed of UART.
+
+Translate the 'Pin' into number for devicetree by using the following formula::
+
+   pin_no = b\*32 + a
+
+where ``a`` and ``b`` are from the Pin value in the table (Pb.a).
+For example, for P0.1, ``pin_no = 1`` and for P1.0, ``pin_no = 32``.
```

Comment:
this is about to be replaced by pinctrl, documented in the nrfx pinctrl bindings

```suggestion
```